example/index.html

```diff
@@ -85,7 +85,7 @@
     <footer data-scroll data-scroll-class="floating" data-scroll-start="#div2" data-scroll-position="top">Footer</footer>
     <script src="../dist/scroll-triggers.js"></script>
     <script>
-      ScrollTriggers([{
+      ScrollTriggers.default([{
```

Comment:
not sure I like this. What are the alternatives?

Reply:
Not export events.

Reply:
What about adding Events to the init object?  So it's ScrollTriggers.Events?

Reply:
Good idea